Lib/selectors.py

```diff
@@ -371,6 +373,33 @@ def unregister(self, fileobj):
             pass
         return key
 
+    def modify(self, fileobj, events, data=None):
+        try:
+            key = self._fd_to_key[self._fileobj_lookup(fileobj)]
+        except KeyError:
+            raise KeyError(f"{fileobj!r} is not registered") from None
+
+        changed = False
+        if events != key.events:
+            selector_events = 0
+            if events & EVENT_READ:
+                selector_events |= self._EVENT_READ
+            if events & EVENT_WRITE:
+                selector_events |= self._EVENT_WRITE
+            try:
+                self._selector.modify(key.fd, selector_events)
+            except BaseException:
```

Comment:
Why `BaseException`? `_PollLikeSelector.register()` uses `Exception`.

If `BaseException` is used intentionally, it can be omitted, the bare `except` has the same effect.